### PR TITLE
Bump up Chronos 3.x.

### PR DESCRIPTION
**Pull request by ADmad**
While Chronos 3 is still WIP it's better to start testing with it early to avoid surprises later.

### CODE REVIEW

File: src/I18n/DateTime.php [hunk 1]
```diff
@@ -126,11 +128,14 @@ class DateTime extends Chronos implements I18nDateTimeInterface, Stringable
      *
      * @param \DateTimeInterface|string|int|null $time Fixed or relative time
      * @param \DateTimeZone|string|null $tz The timezone for the instance
+     * @psalm-immutable
      */
     public function __construct(DateTimeInterface|string|int|null $time = null, DateTimeZone|string|null $tz = null)
     {
         if ($time instanceof DateTimeInterface) {
+            /** @psalm-suppress ImpureMethodCall */
```
Comment (othercorey) [inline src/I18n/DateTime.php:136]: This implicit pure stuff will get annoying.

Comment (ADmad) [reply src/I18n/DateTime.php:136]: Yeah, the error doesn't make sense to me for methods which return types other than the class itself.

Comment (othercorey) [reply src/I18n/DateTime.php:136]: Can we disable purity checks entirely?

Comment (ADmad) [reply src/I18n/DateTime.php:136]: Might be, I can check later.

**Pull request merged**